### ISSUE TITLE
Generates directory B3DM

### ISSUE THREAD

**Issue by quannh997** [fanvanzh/3dtiles#182]
Dear fanvanzh!

I looking for 2 month to find a software or command line to tiles my project 3d, i found out CesiumLab but result not as my expected. Don't you know anything can tiles b3dm into directory with LOD and level zoom like this?
![189133720_382969373100936_1596775566561209629_n](https://user-images.githubusercontent.com/61110829/120450276-848b7f00-c3ba-11eb-9442-7fd407b90524.png)
![190230530_1150635722028233_1008638736973091092_n](https://user-images.githubusercontent.com/61110829/120450280-85bcac00-c3ba-11eb-9675-88b0a5bcf6f0.png)
Thanks!

**Comment by fanvanzh** [comment]
这两个图片是什么关系？
你是想组织 b3dm 文件成 3dtile 的结构吗？

**Comment by quannh997** [comment]
> 这两个图片是什么关系？
> 你是想组织 b3dm 文件成 3dtile 的结构吗？

Yes, what i want is organize b3dm files into a 3dtile structure, most convenient is structure like gdal2tiles

**Comment by fanvanzh** [comment]
那只要写 tileset.json 就可以了，你可以参考 tileset.json 的生成过程。
自己定义每个 child 的 matrix、uri、geometricError 等参数。
